### PR TITLE
Update Workflows

### PR DESCRIPTION
**Pull request by zkamvar**
update workflows for universe release (to be merged after release)

### CODE REVIEW

**Review by elray1 — APPROVED** [review]
lgtm

File: README.Rmd [hunk 1]
```diff
@@ -28,9 +29,19 @@ This package is part of the [hubverse](https://hubverse.io/en/latest/) ecosystem
 
 ## Installation
 
-You can install the development version of hubExamples from [GitHub](https://github.com/) with:
+### Latest
+
+You can install the [latest version of hubExamples from the R-universe](https://hubverse-org.r-universe.dev/hubAdmin):
```
Comment (elray1) [inline README.Rmd:34]: should this link be pointing to hubExamples?

File: README.md [hunk 2]
```diff
@@ -10,19 +10,31 @@ experimental](https://img.shields.io/badge/lifecycle-experimental-orange.svg)](h
...
+ecosystem, which aims to provide a set of tools for infectious disease
+modeling hubs to share and collaborate on their work.
 
 ## Installation
 
-You can install the development version of hubExamples from
+### Latest
+
+You can install the [latest version of hubExamples from the
+R-universe](https://hubverse-org.r-universe.dev/hubAdmin):
```
Comment (elray1) [inline README.md:28]:
```suggestion
You can install the [latest version of hubExamples from the
R-universe](https://hubverse-org.r-universe.dev/hubExamples):
```

**Review by elray1 — APPROVED** [review]
approve subject to readme update

File: README.Rmd (hunk 1 above)
Comment (zkamvar) [reply README.Rmd:34]:
```suggestion
You can install the [latest version of hubExamples from the R-universe](https://hubverse-org.r-universe.dev/hubExamples):
```

**Pull request merged**